<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="4" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="60727" started-at="2017-09-24T16:17:55Z" finished-at="2017-09-24T16:18:56Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="60727" started-at="2017-09-24T16:17:55Z" finished-at="2017-09-24T16:18:56Z">
      <class name="Scripts.ScheduleVisit">
        <test-method status="PASS" signature="login()[pri:1, instance:Scripts.ScheduleVisit@201a4587]" name="login" duration-ms="36511" started-at="2017-09-24T21:47:55Z" finished-at="2017-09-24T21:48:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="search_select()[pri:2, instance:Scripts.ScheduleVisit@201a4587]" name="search_select" duration-ms="6497" started-at="2017-09-24T21:48:32Z" finished-at="2017-09-24T21:48:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- search_select -->
        <test-method status="PASS" signature="filter_select()[pri:3, instance:Scripts.ScheduleVisit@201a4587]" name="filter_select" duration-ms="5865" started-at="2017-09-24T21:48:38Z" finished-at="2017-09-24T21:48:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- filter_select -->
        <test-method status="FAIL" signature="visit()[pri:4, instance:Scripts.ScheduleVisit@201a4587]" name="visit" duration-ms="11824" started-at="2017-09-24T21:48:44Z" finished-at="2017-09-24T21:48:56Z">
          <exception class="java.lang.IndexOutOfBoundsException">
            <message>
              <![CDATA[Index: 1, Size: 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IndexOutOfBoundsException: Index: 1, Size: 0
	at java.util.ArrayList.rangeCheck(ArrayList.java:653)
	at java.util.ArrayList.get(ArrayList.java:429)
	at Scripts.ScheduleVisit.visit(ScheduleVisit.java:128)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IndexOutOfBoundsException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- visit -->
      </class> <!-- Scripts.ScheduleVisit -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
